### PR TITLE
feature: tags functionality in warrants

### LINKED ISSUE #76: feature: Tags functionality in warrants

### PR DESCRIPTION
**Pull request by cdxker**
Good baseline for text based shortcuts

- feat: Tags in warrants displayed, need styling and shortcuts
- feat: Prevents tabbing out of input
- feat: Modified spacings and added X button to remove tags
- bugfix: No longer considers selected items for tab complete
- feat: When outer "input box" is clicked, inner "input box" is focused
- bugfix: changed order that tags are displayed in create WarrantRebuttal form

https://imgur.com/doI9OPH

### CODE REVIEW

File: src/components/Atoms/Combobox.tsx [hunk 1]
```diff
@@ -30,6 +30,8 @@ export const Combobox = (props: ComboboxProps) => {
   const [usingPanel, setUsingPanel] = createSignal(false);
   const [inputValue, setInputValue] = createSignal("");
 
+  let inputBox: HTMLInputElement | undefined = undefined;
```
Comment (skeptrunedev) [inline src/components/Atoms/Combobox.tsx:34]:
```suggestion
  const inputBox: HTMLInputElement | undefined = undefined;
```

Noticed the linter picked this one up

**Review by skeptrunedev — APPROVED** [review]
I have the one comment on using `const` but otherwise this looks good to go! 

This is awesome!!!!!

File: src/components/Atoms/Combobox.tsx (hunk 1 above)
Comment (cdxker) [reply src/components/Atoms/Combobox.tsx:34]: This doesn't work, I tried this and it doesn't work, It seems that solid redefines it later when you set it as ref in a component

Comment (skeptrunedev) [reply src/components/Atoms/Combobox.tsx:34]: Then let's just add the linter ignore comment. That's an interesting error. I wonder if there's a useref hook equivalent in solid.

Comment (cdxker) [reply src/components/Atoms/Combobox.tsx:34]: This apparently is the equivalent of useRef, personally not a fan of it because it feels like magic

**Pull request merged**